site/content/blog/package-authoring-with-kctrl.md

```diff
@@ -0,0 +1,56 @@
+
+A package author encapsulates, versions and distributes Kubernetes manifests as package for package consumers to install on a Kubernetes cluster. They can choose to create a package by using a third party manifest, e.g. they can choose to create a package from [cert-manager](https://cert-manager.io/), [Dynatrace](https://www.dynatrace.com/), etc., or they can distribute their own project Kubernetes manifest by creating a package.
+
+
+## Typical Package Authoring journey
+Package Authoring is an iterative process and below are the most common steps performed by the authors:
+1. Authors know about the Kubernetes manifest they want to package.
+2. Add/change the manifest by adding additional [overlay](https://carvel.dev/ytt/docs/latest/ytt-overlays/)/[template](https://carvel.dev/ytt/docs/latest/#templating) and test the package. This is the iterative part where authors want to make the changes and test them quickly.
+3. Once all the manifest are in place, create the [imgpkg](https://carvel.dev/imgpkg/) bundle (to be mentioned in the package) and the package itself.
+4. Add the package to the package repository for distribution.
+
+## How does `kctrl` help package authors?
+Today, package authors are supposed to know tools like [imgpkg](https://carvel.dev/imgpkg/), [kbld](https://carvel.dev/kbld/), [vendir](https://carvel.dev/vendir/), etc. before they start on the package authoring journey. These Carvel tools has a learning curve of itself. We wanted to introduce a set of CLI commands that guide users in performing most common packaging steps so that they are able to create and release the package without knowing these tools.
+
+{{< asciinema key="authoring-commands-blog" rows="30" preload="1" speed="2">}}
+
+## What are `kctrl` package authoring commands?
+* [**kctrl package init**](/kapp-controller/docs/latest/authoring-command/#initialising-the-package): To initialize the Package, PackageInstall and other resources. These resources will be used by the subsequent commands.
+
+* [**kctrl dev**](/kapp-controller/docs/latest/authoring-command/#dev): This command will use the Package and PackageInstall generated above and deploy them locally. By locally, we mean that kapp-controller need not be installed on the Kubernetes cluster. Also, it will eliminate the need to push the imgpkg bundle to an OCI registry during the development stage. This will be useful when you are develping the additional overlay/template.
+
+* [**kctrl package release**](/kapp-controller/docs/latest/authoring-command/#releasing-the-package): This command will create and upload the imgpkg bundle with all the Kuberentes manifest of the software. Also, it will create the `package.yml` and `metadata.yml` which can be either checkin to the package repository or released as part of the release artifacts.
```

Comment:
```suggestion
* [**kctrl package release**](/kapp-controller/docs/latest/authoring-command/#releasing-the-package): This command will create and upload the imgpkg bundle with all the Kubernetes manifests of the software. Also, it will create the `package.yml` and `metadata.yml` files which can be either checked in to the package repository or released as part of the release artifacts.
```

Reply:
Unresolving this, I would say it sorta blocks ^